.buildkite/scripts/steps/fips/build_cloud_fips_image.sh

```diff
@@ -33,8 +33,8 @@ node scripts/build \
   --skip-docker-serverless \
   --skip-docker-contexts
 
-CLOUD_FIPS_IMAGE=$(docker images --format "{{.Repository}}:{{.Tag}}" docker.elastic.co/kibana-ci/kibana-cloud-fips)
-cat <<EOF | buildkite-agent annotate --style "info" --context kibana-cloud-fips-image
+CLOUD_FIPS_IMAGE=$(docker images --format "{{.Repository}}:{{.Tag}}" docker.elastic.co/kibana-ci/kibana-fips)
```

Comment:
Does this also need to be updated?  https://github.com/elastic/kibana/blob/58014d189d2552de0aa349540d5e2ef6a8369394/src/dev/build/tasks/os_packages/create_os_package_tasks.ts#L162

Due to https://github.com/elastic/kibana/blob/58014d189d2552de0aa349540d5e2ef6a8369394/src/dev/build/tasks/os_packages/docker_generator/run.ts#L58